sa/sa_test.go

```diff
@@ -2633,9 +2633,9 @@ func TestIncidentsForSerial(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to insert disabled incident")
 
 	// No incidents are enabled, so this should return in error.
-	incidentsForSerial, err := sa.IncidentsForSerial(context.Background(), &sapb.Serial{Serial: "1337"})
+	result, err := sa.IncidentsForSerial(context.Background(), &sapb.Serial{Serial: "1337"})
 	test.AssertErrorIs(t, err, berrors.NotFound)
-	test.Assert(t, len(incidentsForSerial) == 0, "There should be 0 incidents")
+	test.Assert(t, result == nil, "Expected nil result")
```

Comment:
use `test.AssertNil(t, result)`, not `test.Assert(t, result == nil)`, here and below